### ISSUE TITLE
listAuditsAction() is missing disabled entities

### ISSUE THREAD

**Issue by webmasterMeyers** [DamienHarper/DoctrineAuditBundle#58]
when viewing `example.com/audit` and entities with config `enabled: false` do not show up in the list.

ex;
````yaml
# config/packages/dh_doctrine_audit.yaml
dh_doctrine_audit:
    entities:
        App\Entity\ContactAddress:
            enabled: false
````
Contact Address will be omitted from the list even though you may have enabled audit at run time

**Comment by DamienHarper** [comment]
@webmasterMeyers I'll have a look at this, thanks for the report

**Comment by DamienHarper** [comment]
@webmasterMeyers this is fixed on `master` branch. It will be available in upcoming 2.2.0 release.

**Comment by DamienHarper** [comment]
@webmasterMeyers 2.2.0 is released
